form/form_collections.rst

```diff
@@ -85,7 +85,7 @@ objects::
 Then, create a form class so that a ``Tag`` object can be modified by the user::
 
     // src/Form/Type/TagType.php
```

Comment:
If you compare this path against the namespace it looks OK (I mean not wrong), but one could argue, that using `Form\Type` might be a Best Practice, which is also 👍

Did you mean that?

Reply:
Yes, I was wrong. Now checked again. I did not see the file path and created file same builder `php bin/console make:form` without `type` folder.

I'm sorry I took up your time.

Reply:
No need to be sorry, I think we should change the path *and* the namespace to not confuse people using the `make:form` command.

What do you think @javiereguiluz ?

Reply:
Looking at the other chapters this seems to be the only place where (still) use the `Type` subnamespace. So I agree we should change it here.

Reply:
@OskarStark we've discussed about this multiple times in the past. There are two confronted opinions: @HeahDude proposes to always use the `Type` sub-namespace for form types; @weaverryan proposes to *not* do that, unless the app is complex and you have multiple "form things" (extensions, types, data mappers, etc.)

I mostly agree with Ryan ... but adding `Type/` wouldn't be such a big deal. It's just one tiny sub-namespace and folder.

Anyway, as @xabbuh said, we don't add `Type/` anywhere else, so let's remove it here too.

Reply:
Thank you for the explanation Javier, I mostly agree with Ryan here, too, because the documentation would not show such complexity in most cases.